lib/components/molecules/CookieDisclaimer/CookieDisclaimer.style.js

```diff
@@ -0,0 +1,37 @@
+import { css } from 'styled-components';
+
+const disclaimerStyles = css`
+  display: flex;
+  flex-flow: column;
+  border: 1px solid grey;
+  padding: ${props => props.theme.spacing.UNIT_4};
+  animation: disclaimer-slide-up 0.5s ease-in;
+  position: absolute;
+  top: 2rem;
+  left: 0;
+  --webkit-backface-visibility: hidden;
+  @keyframes disclaimer-slide-up {
```

Comment:
Key frames are already defined as part of theme, look for base/def/animation.js

Reply:
Moved to common place.